### ISSUE TITLE
Installation issue: 'lmdb.h' file not found

### ISSUE THREAD

**Issue by timothyylim** [citp/BlockSci#46]
I'm installing on osx but I don't think it's an OS issue. Following the installation instructions, I managed to get pretty far until the 'make' command here:
```
cd ../../..
mkdir release
cd release
cmake -DCMAKE_BUILD_TYPE=Release ..
make
```

Which resulted in the following error:

```
[  0%] Building CXX object src/blocksci/CMakeFiles/blocksci_static.dir/address/address.cpp.o
In file included from /Users/timothylim/repos/BlockSci/src/blocksci/address/address.cpp:19:
In file included from /Users/timothylim/repos/BlockSci/src/blocksci/index/address_index.hpp:17:
/Users/timothylim/repos/BlockSci/libs/lmdbxx/lmdb++.h:25:10: fatal error: 'lmdb.h' file not found
#include <lmdb.h>      /* for MDB_*, mdb_*() */
         ^
1 error generated.
```

Unfortunately, I'm not proficient in c++ and am having trouble debugging this.

**Comment by timothyylim** [comment]
@hkalodner, I saw in a previous issue that you are using Mac to run BlockSci. I would really appreciate it if you could share some instructions on how to get set up (if they differ at all from the Unbuntu ones).

**Comment by hkalodner** [comment]
Hi @timothyylim,

The lack of mac build instructions is definitely an issue.

I haven't built BlockSci on a clean machine in a while since I've been developing on my mac laptop, but deploying on Ubuntu. From my memory, I was able to install most or all of the needed dependencies using homebrew.

The following should take care of most of the dependencies hopefully.

`brew install lmdb boost openssl libjson-rpc-cpp python3 google-sparsehash cmake`

On my machine, cmake had some trouble finding OpenSSL so I also had to set the environment variable OPENSSL_ROOT_DIR=/usr/local/opt/openssl/ before running cmake.
